deltachat-ios/AppDelegate.swift

```diff
@@ -527,11 +567,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
             if !self.appIsInForeground() {
                 self.dcAccounts.stopIo()
```

Comment:
is this stopIo() on purpose here? graceful shutdown incl. stopIo() is done one line below